torch/quantization/ns/graph_passes.py

```diff
@@ -330,7 +379,7 @@ def load_arg(a):
                 env_c[node_b.name], gm_b, logger_cls, '_ns_logger_b_', name_b)
             # subgraph so far:
             #
-            #       dtype_cast_node --> node_a_copy
+            #       dtype_cast_node --> subgraph_a_copy
```

Comment:
Should the cast be at the input of node_c?

Reply:
correct.  The code does this, but the comment is wrong, thanks for catching.  Will update before landing.